src/pages/api/webhooks/run-jobs/[[...run]].ts

```diff
@@ -73,9 +72,8 @@ export const jobs: Job[] = [
   rewardsDailyReset,
   ...bountyJobs,
   ...Object.values(eventEngineJobs),
-  // ...csamJobs,
   processClubMembershipRecurringPayments,
-  resourceGenerationAvailability,
+  // ...csamJobs,
```

Comment:
Please remember to keep the `resourceGenerationAvailability` jobs
```suggestion
  // ...csamJobs,
  resourceGenerationAvailability,
```